### PR TITLE
Add MAINTAINERS

### PR DESCRIPTION
**Pull request by hogo6002**
Add a maintainers file, sorted alphabetically

### CODE REVIEW

File: infra/MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Adam Korcz <adam@adalogics.com> @AdamKorcz
+Arthur Chan <arthur.chan@adalogics.com> @arthurscchan
```
Comment (oliverchang) [inline infra/MAINTAINERS:2]: Let's just keep this to Adam and David as the primary contributors here.

**Review by oliverchang — COMMENTED** [review]
thanks!

File: infra/MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Adam Korcz <adam@adalogics.com> @AdamKorcz
```
Comment (jonathanmetzman) [inline infra/MAINTAINERS:1]: nit: I think it woiuld be better to use a real format, like CSV. What if someone mistakenly adds maintainer with a middle name.

Comment (hogo6002) [reply infra/MAINTAINERS:1]: Adding a middle name is fine. But yeah, CSV is a better format, I will update it.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Comment by jonathanmetzman** [comment]
One last nit, it's more readable to have spaces between the columns (eg. `Name, Email` instead of `Name,Email`)

**Comment by hogo6002** [comment]
> One last nit, it's more readable to have spaces between the columns (eg. `Name, Email` instead of `Name,Email`)

Since I updated it to a csv file, it would be better to not have an initial space. But it will show as a preview version in file content. https://github.com/google/oss-fuzz/blob/1d544124c48ddb23b0fc57bfa1c3425b21bc9c3a/infra/MAINTAINERS.csv

**Review by oliverchang — APPROVED** [review]

File: infra/MAINTAINERS.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name,Email,Github Username
+Adam Korcz,<adam@adalogics.com>,AdamKorcz
+David Korczynski,<david@adalogics.com>,DavidKorczynski
+Dongge Liu,<donggeliu@google.com>,Alan32Liu
+Holly Gong,<gongh@google.com>,hogo6002
+Jonathan Metzman,<metzman@google.com>,jonathanmetzman
+Oliver Chang,<ochang@google.com>,oliverchang
```
Comment (oliverchang) [inline infra/MAINTAINERS.csv:7]: Last comment: is there any benefit to keeping the "<>" for the email now that we're just using CSV?

**Pull request merged**